Re-enable use of these tools on big-endian machines.

Al Stone <ahs3@redhat.com>

Index: acpica-unix-20151218/source/compiler/aslmain.c
===================================================================
--- acpica-unix-20151218.orig/source/compiler/aslmain.c
+++ acpica-unix-20151218/source/compiler/aslmain.c
@@ -341,18 +341,6 @@ main (
     int                     ReturnStatus = 0;
 
 
-    /*
-     * Big-endian machines are not currently supported. ACPI tables must
-     * be little-endian, and support for big-endian machines needs to
-     * be implemented.
-     */
-    if (AcpiIsBigEndianMachine ())
-    {
-        fprintf (stderr,
-            "iASL is not currently supported on big-endian machines.\n");
-        return (-1);
-    }
-
     AcpiOsInitialize ();
     ACPI_DEBUG_INITIALIZE (); /* For debug version only */
 
